Task_Position_the_Cubes/index.html

```diff
@@ -1,14 +1,19 @@
 <head>
   <title>box problem</title>
   <link rel="stylesheet" href="test.css">
+  <link rel="stylesheet" href="background_style.css">
+</head>
 
 <body>
-  <div class="parent">
-    parent
-  <div class="child-one child">one</div>
-  <div class="child-two child">two</div>
```

Comment:
The opening comment tag on line 11 is not closed with a closing comment tag. It should be <!-- instead of just <!